### ISSUE TITLE
Installation Issues

### ISSUE THREAD

**Issue by BewilderedAgain** [easyphp/easyphp-devserver#200]
There would appear to be a number of issues with a fresh installation on Windows 10.

I had ver 14 Devserver installed used the uninstall and then installed devserver17

No entry was made in the start menu. Not desirable behavior, best to ask user. 
An icon was created on the desktop so at least I could start it. Again, ot desirable behavior, best to ask user. 
I could not get php to run on my test site ( worked fine with v14) . The windows path still had the path to the old v14 php and no path for the new php. That doesn't seem right.  Also, the http.conf file for Apache is missing the loadmdule php..  line, but oddly the sample demo works.

**Comment by BewilderedAgain** [comment]
Trying to run my target site stored in a specified working directory results in code being display so it seems  the request is handled as a file.  

To add to this.  Trying to run phpinfo.php in the installed documentroot produces a blank page.  No errors in logs.

I can add
LoadModule php5_module "C:/Program Files (x86)/EasyPHP-Devserver-17/eds-binaries/php/php5630vc11x86x200403215108/php5apache2_4.dll"
and run the apache server, but it doesn't help, but apache does run

If I try to add
#PHPiniDir "C:/Program Files (x86)/EasyPHP-Devserver-17/eds-binaries/php/php5630vc11x86x200403215108/"
the apache won't run. Again nothing in the error logs.

 So, in conclusion there has to be something badly wrong with the installation files since  it is claimed "Devserver installs a complete, open source and ready-to-use development environment"
